### ISSUE TITLE
I use 'KTS' to segment the video ,but I can't get the result of yours，how can I get the same segmentation

### ISSUE THREAD

**Issue by sunguoquan1005** [TIBHannover/MSVA#9]

**Comment by mpalaourg** [comment]
I think, that the data used in this repo weren't produced by the authors here, but they are the "same" data used in almost every work (produced by [Kaiyang Zhou](https://github.com/KaiyangZhou/pytorch-vsumm-reinforce)). That being said, I tried many times and with different KTS parameters, but couldn't replicate the same shot segmentation.  The data also appeared in [Video Summarization with Long Short-term Memory](https://arxiv.org/pdf/1605.08110.pdf), where in the Supplementary Material they note, and I quote `The resulting intervals are around 5 seconds on average.` Even with this info, I was unable to replicate them, but I wish you a better luck!

**Comment by StevRamos** [comment]
That's right. In my exploration to find the original steps, I found this https://github.com/SinDongHwan/pytorch-vsumm-reinforce/blob/master/utils/generate_dataset.py which is the most similar. I relied on that last one to replicate and I think I did. At least, in the shape of the vectors they are similar (TVSum and Summe) but the "points of change" depend on the vector of characteristics that was applied, so I got different values. This also happens with the gtscore. This is my repository https://github.com/StevRamos/video_summarization

**Comment by sunguoquan1005** [comment]
> That's right. In my exploration to find the original steps, I found this https://github.com/SinDongHwan/pytorch-vsumm-reinforce/blob/master/utils/generate_dataset.py which is the most similar. I relied on that last one to replicate and I think I did. At least, in the shape of the vectors they are similar (TVSum and Summe) but the "points of change" depend on the vector of characteristics that was applied, so I got different values. This also happens with the gtscore. This is my repository https://github.com/StevRamos/video_summarization

Thank you,I tried that

**Comment by sunguoquan1005** [comment]
> I think, that the data used in this repo weren't produced by the authors here, but they are the "same" data used in almost every work (produced by [Kaiyang Zhou](https://github.com/KaiyangZhou/pytorch-vsumm-reinforce)). That being said, I tried many times and with different KTS parameters, but couldn't replicate the same shot segmentation. The data also appeared in [Video Summarization with Long Short-term Memory](https://arxiv.org/pdf/1605.08110.pdf), where in the Supplementary Material they note, and I quote `The resulting intervals are around 5 seconds on average.` Even with this info, I was unable to replicate them, but I wish you a better luck!

Thank you,I tried that
